Documentation/manual/kcc-design/customization-user-story.md

```diff
@@ -0,0 +1,138 @@
+**Expanding on ‘Customization User’ Scenarios**
+
+**Example #1: Planar 3D Platformer Character Controller**
+
+————————————————————————————————————————————————————————————————————————
+
+User Story:
+
+As a Customization User with an existing project, I’m interested in having my development studio switch to OpenKCC from a paid asset for the following reasons
```

Comment:
I think as a requirement for linting the markdown flies with `markdownlint` I have a constraint that lines shouldn't be more than 80 characters long. The markdown preview will auto join lines that don't have a newline between them. 

You can install an extension in vscode to view the errors in style rules and make correcting them easier :) 

Name: markdownlint
Id: DavidAnson.vscode-markdownlint
Description: Markdown linting and style checking for Visual Studio Code
Version: 0.49.0
Publisher: David Anson
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint